src/Athens-Examples/AthensTextRenderManualTests.class.st

```diff
@@ -136,7 +137,7 @@ AthensTextRenderManualTests class >> test5 [
 			canvas pathTransform translateBy: 10@0.1.
 		]
 	].
-	Display getCanvas translucentImage: surf asForm at: 100@100
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?